tests/bitcoin/BitcoinProcessor.spec.ts

```diff
@@ -85,12 +84,16 @@ describe('BitcoinProcessor', () => {
       height = randomNumber();
     }
     const hash = randomString();
+    const feePaidRandom = randomNumber();

```

Comment:
Design: I do not think `randomNumber()` should have a default max unless the default is MAX_SAFE_INTEGER, not asking to change existing tests, but can we explicitly specify the max for all new `randomNumber()` call?

Reply:
Changing the further usage of the feePaidRandom to not use -1 as to remove any confusion.